src/test/java/com/google/javascript/clutz/ProgramSubject.java

```diff
@@ -108,8 +110,11 @@ private String parse() throws AssertionError {
       roots.add("main.js");
     }
 
+    List<SourceFile> externs = withExterns ? DeclarationGenerator.getDefaultExterns(opts) : NO_EXTERNS;
```

Comment:
nit: ternary followed by condition is somewhat odd, could you just fold the ternary into the conditional?